quickwit-proto/src/lib.rs

```diff
@@ -36,6 +36,8 @@ impl From<SearchStreamRequest> for SearchRequest {
             end_timestamp: item.end_timestamp,
             max_hits: 0,
             start_offset: 0,
+            sort_by_field: None,
```

Comment:
why not keeping the mini dsl on the grpc layer? I think it's better to keep it as we may handle multiple sort fields in the future. For now it's ok, so let's keep it like this.